com.ultimaker.cura.yml

```
@@ -0,0 +1,285 @@
+      - make -j`nproc`
+      - make install
+    sources:
+      - type: archive
+        url: https://www.riverbankcomputing.com/static/Downloads/sip/4.19.25/sip-4.19.25.tar.gz
+        sha256: b39d93e937647807bac23579edbff25fe46d16213f708370072574ab1f1b4211
+
+  - name: pyqt
+    buildsystem: simple
+    build-commands:
+      - python3 configure.py --no-designer-plugin --no-tools --no-qml-plugin --sysroot /app --sip /app/bin/sip --sip-incdir /app/include --confirm-license --assume-shared --disable QtSensors --disable QtSerialPort --disable QtMultimedia --disable QtWebChannel --disable QtLocation -verbose QMAKE_CFLAGS_RELEASE='-I/usr/include/python3.9' QMAKE_CXXFLAGS_RELEASE='-I/usr/include/python3.9'
+      - make -j`nproc`
+      - make install DESTDIR=/app
+    sources:
+      - type: archive
+        url: https://files.pythonhosted.org/packages/28/6c/640e3f5c734c296a7193079a86842a789edb7988dca39eab44579088a1d1/PyQt5-5.15.2.tar.gz
+        sha256: 372b08dc9321d1201e4690182697c5e7ffb2e0770e6b4a45519025134b12e4fc
+
+  - name: nlopt
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: git
+        url: https://github.com/stevengj/nlopt.git
+        commit: 41967f1981d82d8495c0b27151a126bc35ae0d96
+        tag: v2.6.2
+
+
+  - name: boostheaders
+    buildsystem: simple
+    build-commands:
+      - cp -r boost /app/include/boost
+    sources:
+      - type: archive
+        url: http://sourceforge.net/projects/boost/files/boost/1.67.0/boost_1_67_0.tar.bz2
+        sha256: 2684c972994ee57fc5632e03bf044746f6eb45d4920c343937a465fd67a5adba
+
+  - name: clipper
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_CXX_FLAGS=-fPIC
+      - -DBUILD_SHARED_LIBS=OFF
+    sources:
+      - type: archive
+        url: https://sourceforge.net/projects/polyclipping/files/clipper_ver6.4.2.zip
+        sha256: a14320d82194807c4480ce59c98aa71cd4175a5156645c4e2b3edd330b930627
+      - type: file
+        url: https://raw.githubusercontent.com/Ultimaker/cura-build-environment/4.12/projects/clipper_cmakelists_patch.txt
+        sha256: 54c430466bcbe745fc85b757c02a63868d26022891cc1b98fb9862e9e20c79fb
+        dest-filename: CMakeLists.txt
+
+  - name: libnest2d
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_CXX_STANDARD=17
+    sources:
+      - type: git
+        url: https://github.com/Ultimaker/libnest2d.git
+        commit: 031e4e61218edd092fbc54fbc6df145287b628ba
+      - type: file
+        url: https://raw.githubusercontent.com/Ultimaker/cura-build-environment/4.12/projects/libnest2d_find_clipper.cmake
+        sha256: 558d26c1071a4cb31c09e618553a3f98de7b782d779989db0582ab5c854a6d5f
+        dest-filename: FindClipper.cmake
+      - type: shell
+        commands:
+          - mv FindClipper.cmake cmake_modules/
```

```suggestion
        dest: cmake_modules/
```

This is much simpler.